src/sagas.js

```diff
@@ -368,6 +396,33 @@ function* fetchVmsByPage (action) {
 }
 
 function* fetchVmsByCount (action) {
+  const actual = Selectors.getOvirtVersion().toJS()
```

Comment:
I like this approach. It's old-school, nothing fancy but clear for everyone.
Good to note for later use: the split is done on higher level of the call tree. Inner code is written in general-enough way without additional special `if`s. 👍